app/Console/Commands/SendToQuasar.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Rogue\Console\Commands;
+
+use Rogue\Models\Post;
+use Rogue\Models\Signup;
+use Illuminate\Console\Command;
+use Rogue\Jobs\SendPostToQuasar;
+use Rogue\Jobs\SendSignupToQuasar;
+
+class SendToQuasar extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'rogue:quasar
+                            {start=0000-00-00 : Include all signups and posts updated on or after this day. Format: YYYY-MM-DD}
+                            {end? : Include signups and posts updated up to but not including this day. Format: YYYY-MM-DD}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Send the given items to Quasar.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        info('rogue:quasar - Starting to send to Quasar');
+
+        $start = $this->argument('start');
+        $end = $this->argument('end');
```

Comment:
will this be defined is nothing is passed in as the `start` param?

Reply:
If only `start` (or no arguments) are passed, then `$end` gets set to `null`